server/issue.go

```diff
@@ -11,7 +11,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/andygrunwald/go-jira"
+	jira "github.com/andygrunwald/go-jira"
```

Comment:
For some reason gofmt is forcing this -- any idea why now all of a sudden? Will it break anything?

Reply:
not at all, I kinda like it